### DISCUSSION TITLE
Trigger Parameters Hint not working in Jupyter notebooks

### DISCUSSION THREAD

**Discussion by GuilhermeFCO** [posit-dev/positron#4638] [Q&A]
I using jupyter notebooks but parameters hint not showing.

I configured the shortcut like vscode.
![positron-config](https://github.com/user-attachments/assets/04534723-4e27-441c-8374-5f84742baf80)

In VSCode working normally, in same notebook.
![code](https://github.com/user-attachments/assets/293b91c6-6543-4569-a30b-7093ca21ab3d)

In Positron don't show.
![positron](https://github.com/user-attachments/assets/03080cae-76ef-48da-a5ad-27b4fe7a027b)

**Comment by seeM** [comment]
Thank you for the feedback! I'm not able to repro this on macOS with a notebook with a single cell:

https://github.com/user-attachments/assets/46861f49-9ad1-43a0-b699-6ff53de6982b

Could you please share more about your setup? What OS are you using? What Positron version are you using (via `Positron > About` menu)? What Python version? Can you share more of the code in this notebook?

**Comment by GuilhermeFCO** [comment]
Yes! 

My setup is using positron without install extensions.
I'm using in Ubuntu 22.04.4 LTS.
Positron Version: 2024.08.0 build 83.
The python version is 3.10.12.

I'm using to work, isn't possible to share the notebook or screenshots. I'm using pyspark, the session of spark is instantiated with a package.

Example: When i'm going read parquet, i use spark.read.parquet, when i'm typing parquet, and a press ctrl + space, shows the function correctly, but in the moment i open the parentheses, hide all popups, and i trigger parameter not work, even thoug i do like the video.

I found another bug, if a open positron in app list, and i want to use a kernel of a venv, don't work. Work only i open the positron in terminal inside the folder of venv.

**Reply by petetronic** [reply]
On Ubuntu 24.04.1 LTS and Positron 2024.08.0 build 83, I was able to trigger Parameter Hints for functions in Python cells in Notebooks. I noticed that 1Password had registered Ctrl+Shift+Space as a system keybinding and needed to disable that in 1Password first. After doing that, Positron was able to respond to Ctrl+Shift+Space to reveal Parameter Hints.
